Project Design/Digital Prototype/Color Palette.xml

```diff
@@ -0,0 +1,11 @@
+<!--?xml version="1.0" encoding="UTF-8"?-->
+<resources>
+  <color name="primaryColor">#9e9e9e</color>
+  <color name="primaryLightColor">#cfcfcf</color>
+  <color name="primaryDarkColor">#707070</color>
+  <color name="secondaryColor">#004c3f</color>
+  <color name="secondaryLightColor">#38786a</color>
```

Comment:
This is the correct color scheme as per the customer requirements. However, we should check tomorrow with the professor if it might be a better idea to pick a more eye-catching secondary color.